### PR TITLE
Swap Rich with Matteo as Node.js Voting member

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/nodejs/TSC/issues/1504

### CODE REVIEW

**Comment by mcollina** [comment]
cc @Trott

**Review by tobie — APPROVED** [review]

**Comment by mcollina** [comment]
I don't understand how the linter complains about this.

File: README.md [hunk 1]
```diff
@@ -169,14 +169,14 @@ Anyone who has been active in the foundation or one of its member projects, as d
...
 - Marco Ippolito ([@marco-ippolito](https://github.com/marco-ippolito))
-- Matteo Collina ([@mcollina](https://github.com/mcollina))
 - Michael Dawson ([@mhdawson](https://github.com/mhdawson))
 - Mike Samuel ([@mikesamuel](https://github.com/mikesamuel))
 - Mohammed Keyvanzadeh ([@VoltrexKeyva](https://github.com/VoltrexKeyva))
 - Nick O'Leary ([@knolleary](https://github.com/knolleary))
 - Nitin Kumar ([@snitin315](https://github.com/snitin315))
 - Paula Paul ([@paulapaul](https://github.com/paulapaul))
 - Rick Markins ([@rxmarbles](https://github.com/rxmarbles))
+- Rich Trott ([@Trott](https://github.com/Trott), UCSF)
```
Comment (tobie) [inline README.md:179]:
```suggestion
- Rich Trott ([@Trott](https://github.com/Trott))
```

**Comment by joeyguerra** [comment]
There's a warning message in the pipeline output:

Dates-and-Reminders.md
 warning Missing newline character at end of file final-newline remark-lint

**Review by timmywil — APPROVED** [review]

**Review by Trott — APPROVED** [review]
LGTM if the linter is happy

**Pull request merged**